Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -22,11 +22,10 @@ class QDialogButtonBox;
 class QRadioButton;
 class QScrollArea;
 class QVBoxLayout;
+class QStackedWidget;
```

Comment:
While you're in here can you change all theses Qt forward declares to use: `QT_FORWARD_DECLARE_CLASS`
```suggestion
QT_FORWARD_DECLARE_CLASS(QStackedWidget)
```